oap-server/server-starter/src/main/resources/otel-oc-rules/k8s-cluster.yaml

```diff
@@ -31,8 +31,6 @@
 expSuffix: tag({tags -> tags.cluster = 'k8s-cluster::' + tags.cluster}).service(['cluster'])
```

Comment:
Should we add a filter for k8s too?

Reply:
> Should we add a filter for k8s too?

I'm not sure... on the one hand, adding a default filter requires the users to give the same job name when they set up OpenTelemetry job, otherwise there is no data and confused for users, on the other hand, add a filter is absolutely right to narrow down the metrics that need to be processed in this file

Reply:
Isn't k8s monitoring using OTEL job(s) to forward data?

Reply:
> we can add `kubernetes-cadvisor` and `kube-state-metrics` cause example used these name
> https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/otel-collector-config.yaml#L33

If I have to add a filter, the names we use in the example are first choice for sure, my question is, should we add those if there is no obvious conflicts at the moment?

Reply:
I would recommend yes. Such as using a `Istio*` to filter all Istio MAL scripts, makes sense always.

Reply:
> I would recommend yes. Such as using a `Istio*` to filter all Istio MAL scripts, makes sense always.

Not sure what you mean. The names of jobs for istio doesn’t necessarily always include `istio*`, the job name is totally human written, it can be arbitrary